qa/TL0_python_self_test_frameworks/test_mxnet.sh

```diff
@@ -5,6 +5,7 @@ target_dir=./dali/test/python
 
 test_body() {
     nosetests --verbose -m '(?:^|[\b_\./-])[Tt]est.*mxnet' test_dltensor_operator.py
+    nosetests --verbose test_external_source_parallel_mxnet.py
```

Comment:
Are you brave enough to add paddle paddle test as well?

Reply:
Oh boy. I can try.

Reply:
Hmm, looks like the regular ExternalSource doesn't have paddle support:
```
        NumPy ndarray (CPU)
        MXNet ndarray (CPU)
        PyTorch tensor (CPU or GPU)
        CuPy array (GPU)
        objects implementing __cuda_array_interface__
        DALI Tensor object
```
I will leave it out of this for now.

Reply:
ok